requirements.txt

```diff
@@ -0,0 +1,28 @@
+atomicwrites==1.1.5
+attrs==18.1.0
+certifi==2018.8.13
+chardet==3.0.4
+click==6.7
+colorama==0.3.9
+coverage==4.5.1
+coveralls==1.3.0
+docopt==0.6.2
+Flask==1.0.2
+Flask-API==1.0
+gunicorn==19.9.0
+idna==2.7
+itsdangerous==0.24
+Jinja2==2.10
+MarkupSafe==1.0
+more-itertools==4.3.0
+nose==1.3.7
```

Comment:
Why would you have two testing frameworks. I see you have both `nose` and `pytest` as dependencies?

Reply:
I have removed pytest